app/Console/Kernel.php

```diff
@@ -25,11 +26,16 @@ protected function schedule(Schedule $schedule)
             ->everySixHours()
             ->sendOutputTo($output_filename);
 
-        $schedule->job(new PruneJobs())
+        $schedule->job(new CleanupDatabase())
+            ->everyFourHours()
+            ->withoutOverlapping()
+            ->onOneServer();
```

Comment:
`->onOneServer()` should be avoided here because we currently use `file` as our cache option.  See the [docs](https://laravel.com/docs/11.x/scheduling#running-tasks-on-one-server) for more information.  That said, we currently only officially support one primary web server, so these commands are only ever run on one server anyway, so it's safe to just drop the `onOneServer` constraint, at least for now.